package.json

```diff
@@ -11,50 +11,50 @@
     "vanilla/*"
   ],
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "lerna": "lerna",
-    "clean": "lerna run --parallel clean",
-    "build": "lerna run --parallel build",
-    "lint": "lerna run --parallel lint:es",
-    "lint:fix": "lerna run --parallel lint:es:fix",
-    "tsc": "lerna run --parallel tsc",
-    "test": "lerna run --parallel test",
+    "clean": "npm run clean --workspaces --if-present",
+    "clean:all": "rm -rf node_modules && npm run clean:all --workspaces --if-present",
+    "build": "npm run build --workspaces --if-present",
```

Comment:
We should look at using `postprepare` in each workspace and remove this altogether.

Reply:
Not opposed to it. Future PR